experiments/interpretation/dutchf3_patch/distributed/train.py

```diff
@@ -137,7 +136,7 @@ def run(*options, cfg=None, local_rank=0, debug=False):
             PadIfNeeded(
                 min_height=config.TRAIN.PATCH_SIZE,
                 min_width=config.TRAIN.PATCH_SIZE,
-                border_mode=cv2.BORDER_CONSTANT,
+                border_mode=0,
```

Comment:
My suggestion would be to to define constants to mean something and place them into __init__.py, for example inside cv_lib. Then import like "from cv_lib import OPENCV_BORDER_CONSTANT" and set "border_mode=OPENCV_BORDER_CONSTANT". Then you know what the constant means - "0" isn't very descriptive.

Reply:
That makes sense, I'll add this to `__init__.py`

Reply:
So it turned out I can't import it from `__init__.py`, but I found a better solution to add it the YACS config file. One  less import to worry about :)